CanvasColorSpaceProposal.md

```diff
@@ -72,43 +72,47 @@ interface CanvasColorSpace {
   const CanvasColorSpaceEnum srgb = "srgb";
   const CanvasColorSpaceEnum displayP3 = "display-p3";
 };
-
-// Feature activation:
-
-partial dictionary CanvasRenderingContext2DSettings {
-  CanvasColorSpaceEnum colorSpace = "srgb";
-};
-
-partial dictionary WebGLContextAttributes {
-  CanvasColorSpaceEnum colorSpace = "srgb";
-};
 </pre>
 
 Example:
 <pre>
 canvas.getContext('2d', { colorSpace: "display-p3"} );,
 </pre>
 
-#### The ``colorSpace`` canvas creation attribute
-
 The ``colorSpace`` attribute specifies the color space for the backing storage of the canvas.
```

Comment:
At this point in the new text, the "colorspace" attribute hasn't been defined in IDL yet. Should there be a mention that it's defined in subsequent sections?

Reply:
You're right.

This is a part that will need re-working. I had started doing bits of clean-up along with the restructuring, but the diffs became unwieldy.

Once this general re-structure in is in place (and once the ImageBitmap part is added), then I'm going to do a final pass of tightening up the verbiage and fixing these sorts of dependencies.

Reply:
OK, fine by me to move forward with this and continue to do cleanups.